automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspacesAzureApiSpec.scala

```diff
@@ -32,25 +37,116 @@ class AzureWorkspacesSpec extends AnyFlatSpec with Matchers with CleanUp {
     Some(UUID.fromString("f41c1a97-179b-4a18-9615-5214d79ba600"))
   )
 
-  implicit val token: AuthToken = ownerAuthToken
+  private val wsmUrl = RawlsConfig.wsmUrl
+
+  implicit val system = ActorSystem()
 
   "Rawls" should "allow creation and deletion of azure workspaces" in {
+    implicit val token = owner.makeAuthToken()
```

Comment:
While trying to figure out why the test was failing when run by the workflow, I moved the implicit into each of the individual test methods. I could probably move it back out, but it also seems OK to leave like this (which is actually how it's done in other test files).